src/OrchardCore/OrchardCore.Users.Core/Extensions/ExternalLoginInfoExtensions.cs

```diff
@@ -0,0 +1,10 @@
+using System.Security.Claims;
+
+namespace Microsoft.AspNetCore.Identity;
+
+public static class ExternalLoginInfoExtensions
+{
+    public static string GetEmail(this ExternalLoginInfo info)
+        => info.Principal.FindFirstValue(ClaimTypes.Email)
+        ?? info.Principal.FindFirstValue("email");

```

Comment:
```suggestion
        => info.Principal.FindFirstValue(ClaimTypes.Email) ?? info.Principal.FindFirstValue("email");
```

Reply:
Vertical over horizontal is the way to go for me.

Reply:
The current formatting suggests that `=>` is on the same level of syntax as `?? `. First, I was scratching my head.

I actually prefer `=>` to be on the line of the method signature since then the body expression can be multi-line without such issues. But whatever, just this particular combination is really confusing (and unusual).